book.rb

```diff
@@ -0,0 +1,14 @@
+require './rental'
+class Book
+  attr_accessor :title, :author, rental
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
+
+  def add_book(person, date)
```

Comment:
- [ ] nice work. This method should be for adding rental and not book. please check that.